### PR TITLE
More descriptive reorg log

### LINKED ISSUE #5427: More descriptive reorg logs
### Describe the feature

Currently we only know whether or not there was a reorg with these metrics:
https://github.com/paradigmxyz/reth/pull/3860

or if there is a log like the following:
```
2023-10-29T23:16:28.076835Z  INFO blockchain_tree: Unwinding canonical chain blocks: 43510..=43510
```

It should be more clear when there was a reorg, and this log should be more easily searchable / greppable.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by fgimenez**
Closes #5427 

Added "Reorg" explicitly to the log message and included words in the function name to make it easier to find/grep.

### CODE REVIEW

**Review by mattsse — APPROVED** [review]
I like this.

wdyt @rkrasiuk ?

File: crates/blockchain-tree/src/blockchain_tree.rs [hunk 1]
```diff
@@ -1242,7 +1242,7 @@ impl<DB: Database, EF: ExecutorFactory> BlockchainTree<DB, EF> {
 
         let tip = provider_rw.last_block_number()?;
         let revert_range = (revert_until + 1)..=tip;
-        info!(target: "blockchain_tree", "Unwinding canonical chain blocks: {:?}", revert_range);
+        info!(target: "blockchain_tree", "Reorg: revert canonical from database by unwinding chain blocks {:?}", revert_range);
```
Comment (Rjected) [inline crates/blockchain-tree/src/blockchain_tree.rs:1245]:
```suggestion
        info!(target: "blockchain_tree", "REORG: revert canonical from database by unwinding chain blocks {:?}", revert_range);
```

File: crates/blockchain-tree/src/blockchain_tree.rs [hunk 2]
```diff
@@ -64,7 +64,7 @@ use tracing::{debug, error, info, instrument, trace, warn};
 /// classDef pending fill:#FFCA3A
 /// classDef sidechain fill:#FF595E
 /// ```
-/// 
+///
```
Comment (Rjected) [inline crates/blockchain-tree/src/blockchain_tree.rs:67]:
```suggestion
/// 
```

Comment (Rjected) [reply crates/blockchain-tree/src/blockchain_tree.rs:67]: this is needed for `cargo fmt`

**Review by Rjected — DISMISSED** [review]
needs a smol change for `cargo fmt`

**Pull request merged**